guides/common/modules/proc_disable-transparent-hugepage.adoc

```diff
@@ -1,5 +1,5 @@
 [id="Disable_Transparent_Hugepage_{context}"]
-= Disable Transparent Hugepage
+= Disable transparent hugepage
```

Comment:
unsure if it's a name or not. :shrug:

Reply:
I'm not sure why this is a procedure. It's more of a concept module.

Reply:
In the meantime, I will capitalize the title so that it is consistent with the text.